### PR TITLE
Add RAILS_ENV=test for dev container

### LINKED ISSUE #45: Dev server should not compile JiT
Use `RAILS_ENV=test`

### PR DESCRIPTION
**Pull request by eckdanny-osi**
Runs the docker container on the dev node using the environmental variable `RAILS_ENV=test`. This way `webpack` will not attempt to compile just in time and will "build" when the rails server starts.

Resolves #45

### CODE REVIEW

**Comment by edgibbs** [comment]
Can one of the admins verify this patch?

File: Jenkinsfile [hunk 1]
```diff
@@ -47,7 +47,7 @@ node('cm-slave') {
             sh "docker ps --all --quiet --filter \"name=cm-latest\" | xargs docker rm"
             withDockerRegistry([credentialsId: DOCKER_REGISTRY_CREDENTIALS_ID]) {
                 sh "docker pull ${DOCKER_GROUP}/${DOCKER_IMAGE}"
-                sh "docker run --detach --publish 80:3000 --name ${DOCKER_CONTAINER_NAME} -e APP_NAME=casemanagement ${DOCKER_GROUP}/${DOCKER_IMAGE}"
+                sh "docker run --detach --publish 80:3000 --name ${DOCKER_CONTAINER_NAME} --env APP_NAME=casemanagement --env RAILS_ENV=test ${DOCKER_GROUP}/${DOCKER_IMAGE}"
```
Comment (eckdanny-osi) [inline Jenkinsfile]: @RamuKammagani-osi and me wondering if we should use `RAILS_ENV=production`. Seems better to catch this stuff as early as possible... we just need to use a `SECRET_KEY_BASE` env var on the server or pass at run time via jenkins. Thoughts?

**Comment by eckdanny-osi** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Comment by edgibbs** [comment]
retest this please

**Pull request merged**